amnestywebsite/humanity-theme
# Replace page-petitions.php with FSE template

Ref: https://github.com/amnestywebsite/humanity-theme/issues/376

**Steps to test**:
1. make sure petitions is enabled, and that you have some published petitions
2. checkout & run a build
3. create a page
4. set the template to "Page Petitions" (see below screenshot)
5. paste the below markup into the editor in code view
6. publish the page
7. view the frontend
8. it should look similar to https://www.amnesty.org/en/get-involved/take-action/ (content will be different)

![](http://bigbite.im/i/AiNX1q+)

```html




<p class="has-text-align-center">Amnesty supporters are making the world a fairer place. Because of the actions of individuals, lives have been saved, unfair laws have changed, the wrongfully imprisoned have been released. </p>



<p class="has-text-align-center"><strong>Your actions make a difference.</strong></p>



<hr class="wp-block-separator has-css-opacity is-style-wide"/>



<h2 class="wp-block-heading has-text-align-center" id="h-take-action-today">TAKE ACTION TODAY</h2>







<hr class="wp-block-separator has-css-opacity is-style-wide"/>



<h2 class="wp-block-heading has-text-align-center" id="h-get-involved-with-our-global-campaigns">GET INVOLVED WITH OUR GLOBAL CAMPAIGNS</h2>




```
